nix/Readme.md

```diff
@@ -0,0 +1,33 @@
+## Tabry Nix Configurations
+
+### Home Manager Module
+
+This repository provides a home-manager (https://github.com/nix-community/home-manager)
+module to make tabry easy to install and use via home manager.
+
+To use the home-manager module via flakes, add this module to your home-manager
+configuration:
+
+```nix
+{
+  inputs = {
+    tabry.url = "github:evanbattaglia/tabry-rs";
+  };
+  outputs = { ..., tabry }: {
+    homeConfigurations.<user> = {
+      modules = [
+        tabry.homeModules.tabry
+        {
+          config.programs.tabry = {
+            enable = true;
+            enableBashIntegration = true;
+            tabryFiles = [
+              ./zellij.tabry
```

Comment:
I should probably figure out a better example, as zellij actually does have its own shell completion... When I do, I'll change it here

One useful example I may make (especially when I get delegation working better) is you could have your own aliases `dc r`, `dc e`, etc. -- one command with subcommands delegated to different commands -- and have shell completion for all the subcommands ... right now I don't actually know an easy way of doing that without tabry.